etc/alternc/templates/proftpd/conf.d/sftp.conf

```diff
@@ -0,0 +1,44 @@
+<VirtualHost 0.0.0.0>
+    SFTPEngine on
+    Port 2222
+    SFTPLog /var/log/proftpd/sftp.log
+
+    SFTPOptions AllowInsecureLogin
+    SFTPAuthMethods password
+
+    # database@host:port login password
+    SQLConnectInfo                  %%dbname%%@%%dbhost%%:3306 %%dbuser%% %%dbpwd%%
+    # Table :
+    SQLUserInfo ftpusers name encrypted_password uid uid homedir NULL
+
+    # Use mysql PASSWORD function
+    SQLAuthTypes                    Crypt
```

Comment:
From the documentation http://www.proftpd.org/docs/contrib/mod_sql.html, it suggests that only the first 8 characters are taken into account in the password. However, in crypt(3) I don't see mention of that limitation. Do you know if this problem actually happens here?